www/docs/adapters/models.md

```diff
@@ -3,60 +3,48 @@ id: models
 title: Models
 ---
 
```

Comment:
I think this would benefit if we added a visual representation of the relations between the models.

Reply:
Added an image of the schema to the model page :+1: